app/services/build_response_pdf_file_service.rb

```diff
@@ -0,0 +1,161 @@
+class BuildResponsePdfFileService # rubocop:disable Metrics/ClassLength
```

Comment:
This was actually moved as part of refactoring but because there were other changes to share more code between the response and claim pdf's - github is seeing it as a new file - its lying.